flink-connectors/flink-connector-rabbitmq/src/main/java/org/apache/flink/streaming/connectors/rabbitmq/common/RMQConnectionConfig.java

```diff
@@ -29,21 +29,24 @@
 import java.security.KeyManagementException;
 import java.security.NoSuchAlgorithmException;
 import java.util.Optional;
+import java.util.concurrent.TimeUnit;
 
 /**
  * Connection Configuration for RMQ. If {@link Builder#setUri(String)} has been set then {@link
  * RMQConnectionConfig#RMQConnectionConfig(String, Integer, Boolean, Boolean, Integer, Integer,
- * Integer, Integer, Integer)} will be used for initialize the RMQ connection or {@link
+ * Integer, Integer, Integer, Long)} will be used for initialize the RMQ connection or {@link
  * RMQConnectionConfig#RMQConnectionConfig(String, Integer, String, String, String, Integer,
- * Boolean, Boolean, Integer, Integer, Integer, Integer, Integer)} will be used for initialize the
- * RMQ connection
+ * Boolean, Boolean, Integer, Integer, Integer, Integer, Integer, Long)} will be used for initialize
+ * the RMQ connection
  */
 public class RMQConnectionConfig implements Serializable {
 
     private static final long serialVersionUID = 1L;
 
     private static final Logger LOG = LoggerFactory.getLogger(RMQConnectionConfig.class);
 
+    private static final long DEFAULT_DELIVERY_TIMEOUT = 30000;
```

Comment:
I'd prefer to have this value as part of the `RMQSource` class. This way this class does not hold any logic and it can be easier reused.

Reply:
@SteNicholas What was your reasoning to put it here?